.vscode/settings.json

```diff
@@ -6,8 +6,8 @@
   "python.linting.pydocstyleEnabled": true,
   "python.linting.enabled": true,
   "python.linting.flake8Args": [
-    "--ignore=E121,E402,F481,E501,F841",
-    "--max-line-length=89"
+    "--max-line-length=89",
+    "--ignore=W503, B903, E402, F541"
```

Comment:
with aligning I meant to use the setup.cfg also here. Basically it would be best to have just one file of truth and not settings everywhere...

Reply:
it is not possible in our case, I will explain you why